language-plutus-core/src/Language/PlutusCore/Rename.hs

```diff
@@ -46,3 +48,17 @@ instance (HasUnique (tyname ann) TypeUnique, HasUnique (name ann) TermUnique) =>
 
 instance Rename a => Rename (Normalized a) where
     rename = traverse rename
+
+newtype Dupable a = Dupable
+    { unDupable :: a
+    } deriving (Show, Eq, Functor, Foldable, Traversable)
+
+instance Applicative Dupable where
```

Comment:
`deriving (Applicative, Monad) via Identity`, one day...

Reply:
We can actually make it 

```haskell
newtype Dupable = Dupable
    { unDupable :: Identity a
    } deriving (Show, Eq, Functor, Foldable, Traversable, Applicative, Monad)
```

since none of the `Dupable` and `unDupable` are exported anyway.

Reply:
Did that. A little bit weird, but concise and sensible, so why not.